.travis.yml

```diff
@@ -1,10 +1,19 @@
 language: csharp
-mono:
-  - 4.8.1
+sudo: required
+dist: trusty
+dotnet: 1.0.0-preview2-003121
 solution: GoogleMapsApi.sln
+
+addons:
```

Comment:
why do you need this? I think that travisCI supports .net core out of the box.
https://docs.travis-ci.com/user/languages/csharp/